app/src/App.css

```diff
@@ -52,3 +52,43 @@
   cursor: pointer;
 }
 
+.App .preHome Button {
+  background-color: #ff9900;
+  color: white;
+  font-size: 16px;
+  display: inline-block;
+  padding: 14px 20px;
+  width: 100%;
+  margin: 8px 0;
+  border: none;
+  cursor: pointer;
+  width: 110px;
+}
+
+/* 
```

Comment:
If this code is not required, please remove out the commented code

Reply:
Done. https://github.com/aws-samples/medical-transcription-analysis/blob/f6051ba5393880e46671671fc967c4d679707992/app/src/App.css#L55-L66